tsconfig.base.json

```
@@ -101,6 +101,7 @@
       "@nx/typedoc-theme": ["typedoc-theme/src/index.ts"],
       "@nx/vite": ["packages/vite"],
       "@nx/vite/*": ["packages/vite/*"],
+      "@nx/vue": ["packages/vue/src/index.ts"],
```

This path is incorrect. There is no `src/index.ts`.
```suggestion
      "@nx/vue": ["packages/vue"],
```